packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1943,6 +1944,51 @@ export class MergeTree {
         }
     }
 
+    /**
+     * Perform local rollback of previously deleted segments
+     */
+    public rollbackRemovedSegment(segments: ISegment[]) {
```

Comment:
With this rollback functionality in mergeTree, the functionality for insert and annotate should probably be moved to mergeTree as well to match.